sahi/utils/file.py

```diff
@@ -62,7 +62,7 @@ def load_json(load_path):
         load_path: "dirname/coco.json"
```

Comment:
It should be better as:

```python
def load_json(load_path: str, encoding: str = "utf-8"):
    """
    Loads json formatted data (given as "data") from load_path
    Encoding type can be specified with 'encoding' argument.

    Example inputs:
        load_path: "dirname/coco.json"
    """
    # read from path
    with open(load_path, encoding=encoding) as json_file:
        data = json.load(json_file)
    return data
```

Reply:
Got it! Thanks! The changes are updated.